src/metabase/driver/query_processor.clj

```diff
@@ -92,6 +92,12 @@
                                                   msg)))
                                             explanation))))
 
+
+;; +-------------------------------------------------------------------------------------------------------------------+
```

Comment:
sure, i'm happy to do that.  i literally copied that from other places in the code such as https://github.com/metabase/metabase/blob/master/src/metabase/driver/query_processor.clj